src/components/ThoughtAnnotation.js

```diff
@@ -20,12 +19,16 @@ import {
   pathToContext,
   unroot,
 } from '../util'
+import { store } from '../store'
```

Comment:
We don't need to import store here. The first argument of mapStateToProps is `state`, which is same as store.getState()

Reply:
@anmolarora1 Actually we need store here, I have replaced store call in `mapStateToProps` but we need a store in `ThoughtAnnotation`.